projects/go-lib/src/lib/components/go-icon-button/go-icon-button.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@tangoe/gosheets/base/variables';
+@import '~@tangoe/gosheets/base/mixins';
+
+.go-icon-button {
+  @include transition(background-color);
+
+  border: 0;
+  border-radius: $global-radius--round;
+  color: $theme-light-color;
+  cursor: pointer;
+  display: inline-flex;
+  box-shadow: $theme-light-border;
```

Comment:
I don't think this would generate a correct property value. I think you want `$global-box-shadow` here!